mllm/backends/cpu/kernels/common/fa2_1/fwd_bshd.hpp

```
@@ -0,0 +1,91 @@
+#if defined(MLLM_HOST_ARCH_ARM64) || defined(MLLM_HOST_ARCH)
+#include "mllm/backends/cpu/kernels/common/fa2_1/impl-arm.hpp"
+#else
+#include "mllm/backends/cpu/kernels/common/fa2_1/impl-any-simd.hpp"
+#endif
+#include "mllm/backends/cpu/kernels/common/fa2_1/impl-any.hpp"
+
+namespace mllm::cpu::flash_attn2 {
+
+// QKV is BSHD
+// H_KV should <= H_Q
+template<typename __ArchTag, typename __QDType, typename __KDType, typename __VDType, typename __ODType, typename __AccDType,
```

Clang (14.0.6)

[error] declaration uses identifier '__ArchTag', which is a reserved identifier

(bugprone-reserved-identifier,-warnings-as-errors)